tests/e2e/flags/flags.go

```diff
@@ -26,7 +26,7 @@ var Opts = Flags{
 	},
 	KeepExportData: false,
 	InstallCilium:  true,
-	CiliumVersion:  "v1.12.6",
+	CiliumVersion:  "v1.15.1",
```

Comment:
#renovateit We can at least add a note to [Renovate's draft PRs](https://github.com/cilium/tetragon/blob/31117d0596337b6219360ae813ae3c9393991a52/.github/renovate.json5#L393) to make sure it's updated I guess?